py/desispec/quicklook/quicklook.py

```diff
@@ -260,12 +260,16 @@ def runpipeline(pl,convdict,conf,mergeQA=False):
     qlog=qllogger.QLLogger("QuickLook",0)
     log=qlog.getlog()
     passqadict=None #- pass this dict to QAs downstream
-
+    StepsArr=[]
+    mergedQADict={'NIGHTS':[{'NIGHT':conf['Night'],'EXPOSURES':[{'EXPID':conf['Expid'],'FLAVOR':conf['Flavor'],'CAMERAS':[{'CAMERA':conf['Camera'],'PIPELINE_STEPS':StepsArr}]}]}]}
```

Comment:
Not the easiest code to read, but I suspect it is fine.

Reply:
This line is providing the schema for the top level merged dictionary.  It should be directly related to the structure used for offline.

Reply:
@profxj Attached please find merged QA files for dark science, and arc lamp exposure processing.  They show the structure produced by this branch.  Please note that the order of the keys is scrambled in the yaml when it is read, but the format closely follows the discussed data model refinements.  

[mergedQA.tar.gz](https://github.com/desihub/desispec/files/1415439/mergedQA.tar.gz)